src/PrestaShopBundle/Form/Admin/Type/TypeaheadProductCollectionType.php

```diff
@@ -40,6 +41,7 @@ class TypeaheadProductCollectionType extends CommonAbstractType
 {
     protected $productAdapter;
     protected $categoryAdapter;
+    private $configuration;
```

Comment:
any reason to use private here?

Reply:
Yea, no reason to use private here changed to protected.